### PR TITLE
converted ZClientViewController.mediaPlaybackManager callers and ConversationContentViewControllerDelegate to Swift

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

converted `ZClientViewController.mediaPlaybackManager` and `ConversationContentViewControllerDelegate` callers to Swift

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+ConversationContentViewControllerDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            NotificationCenter.default.addObserver(self, selector: #selector(menuDidHide(_:)), name: UIMenuController.didHideMenuNotification, object: nil)
+            
+            return false
+        }
+        
+        return true
+    }
+    
+    func conversationContentViewController(_ contentViewController: ConversationContentViewController, performImageSaveAnimation snapshotView: UIView?, sourceRect: CGRect) {
+        if let snapshotView = snapshotView {
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+ConversationContentViewControllerDelegate.swift]: Could unwrap the snapshot view or return here:

`guard let snapshotView = snapshotView else { return }`

Comment (typfel) [reply Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+ConversationContentViewControllerDelegate.swift]: Or change the method not to accept an optional `UIView`

Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+ConversationContentViewControllerDelegate.swift]: I checked that the caller may pass a nil snapshotView so I make it optional.

Comment (typfel) [reply Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+ConversationContentViewControllerDelegate.swift]: Right but the method does nothing if the view is nil we could just as well not allow it.

**Pull request merged**